File: DesignHashmap/DesMap.py
# Не забывать self, из за ООП в задаче
class MyHashMap:
    def __init__(self):
            self.hash_table = [-1]*1000001
    def put(self, key: int, value: int) -> None:
            self.hash_table[key] = value
    def get(self, key: int) -> int:
            return self.hash_table[key]
    def remove(self, key: int) -> None:
            self.hash_table[key] = -1
        
obj = MyHashMap()
obj.put(5, 5)
param_2 = obj.get(5)
obj.remove(5)

File: DesignHashmap/DesMap.js
// ! Не забывать this, из за ООП в задаче
var MyHashMap = function () {
  this.hashMap = {};
};

/**
 * @param {number} key
 * @param {number} value
 * @return {void}
 */
MyHashMap.prototype.put = function (key, value) {
  this.hashMap[key] = value;
};

/**
 * @param {number} key
 * @return {number}
 */
MyHashMap.prototype.get = function (key) {
  return Object.prototype.hasOwnProperty.call(this.hashMap, key)
    ? this.hashMap[key]
    : -1;
};

/**
 * @param {number} key
 * @return {void}
 */
MyHashMap.prototype.remove = function (key) {
  delete this.hashMap[key];
};

/**
 * Your MyHashMap object will be instantiated and called as such:
 * var obj = new MyHashMap()
 * obj.put(key,value)
 * var param_2 = obj.get(key)
 * obj.remove(key)
 */
